clients/common/norm.cpp

```diff
@@ -171,6 +171,27 @@ double norm_check_general<hipblasBfloat16>(
     return norm_check_general<float>(norm_type, M, N, lda, hCPU_double, hGPU_double);
 }
 
+template <>
+double
+    norm_check_general<int32_t>(char norm_type, int M, int N, int lda, int32_t* hCPU, int32_t* hGPU)
+{
+    // norm type can be 'M', 'I', 'F', 'l': 'F' (Frobenius norm) is used mostly
+
+    host_vector<float> hCPU_double(N * lda);
+    host_vector<float> hGPU_double(N * lda);
+
+    for(int i = 0; i < M; i++)
+    {
+        for(int j = 0; j < N; j++)
+        {
+            hCPU_double[i + j * lda] = (hCPU[i + j * lda]);
+            hGPU_double[i + j * lda] = (hGPU[i + j * lda]);
+        }
+    }
```

Comment:
Does it have to be `hCPU_float`, `hGPU_float` instead of `hCPU_double`, `hGPU_double`?

Reply:
Yep! Fixed.